packages/aws-cdk-lib/aws-cloudwatch/lib/alarm.ts

```diff
@@ -137,6 +163,16 @@ export class Alarm extends AlarmBase {
     return new Import(scope, id);
   }
 
+  /**
+   * Determine whether this operator is an anomaly detection operator.
+   *
+   * @param operator the comparison operator for the alarm.
+   * @returns true if the operator is an anomaly detection operator, false otherwise.
+   */
+  public static isAnomalyDetectionOperator(operator?: ComparisonOperator): boolean {
```

Comment:
Does this function need to be public?

I would just make it a module-local free-floating function.